src/encoded/types/experiment.py

```diff
@@ -142,6 +144,12 @@ class Experiment(Item):
         "files.badges.badge.description",
         "files.badges.badge.badge_icon",
         "files.badges.messages",
+        "files.quality_metric.display_title",
```

Comment:
you could delete this line ^^ because display_title always gets embedded when other properties are embedded.